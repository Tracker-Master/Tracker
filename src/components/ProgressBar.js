import React from 'react';
import { StyledProgress } from 'Styles/components/ProgressBarStyles';

const ProgressBar = () => {
  return (
    <StyledProgress>
      <progress max="100" value="69.56">
        69.56%
      </progress>
    </StyledProgress>
  );
};

export default ProgressBar;
