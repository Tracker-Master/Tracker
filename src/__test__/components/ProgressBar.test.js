import React from 'react';
import { shallow } from 'enzyme';
import { create } from 'react-test-renderer';
import ProgressBar from 'Components/ProgressBar';

describe('<ProgressBar />', () => {
  
  test('ProgressBar rendering', () => {
    const progressBar = shallow(<ProgressBar />);
    expect(progressBar.length).toBe(1);
  });

  test('ProgressBar snapshot', () => {
    const wrapper = create(<ProgressBar />);
    expect(wrapper).toMatchSnapshot();
  });

});