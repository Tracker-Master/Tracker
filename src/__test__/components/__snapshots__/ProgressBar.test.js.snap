// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ProgressBar /> ProgressBar snapshot 1`] = `
.c0 {
  margin: 8px auto 50px;
}

.c0 progress {
  width: 100%;
  height: 18px;
  -webkit-appearance: none;
  -moz-appearance: none;
  appearance: none;
  -webkit-appearance: none;
  -moz-appearance: none;
  border: 1px solid rgba(103,58,183,0.75);
  border-radius: 30px;
  overflow: hidden;
}

.c0 progress::-webkit-progress-bar {
  background-color: transparent;
}

.c0 progress::-webkit-progress-value {
  background: linear-gradient( to right,#764BA2,#667EEA );
}

@media (min-width:768px) {

}

@media (min-width:1024px) {

}

<div
  className="c0"
>
  <progress
    max="100"
    value="69.56"
  >
    69.56%
  </progress>
</div>
`;
